fastlane-plugin-wpmreleasetoolkit.gemspec

```
@@ -37,6 +37,10 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'parallel', '~> 1.14'
   spec.add_dependency 'chroma', '0.2.0'
   spec.add_dependency 'activesupport', '~> 5'
+
+  # `google-cloud-storage` is required by fastlane, but we pin it in case it's not in the future
+  spec.add_dependency 'google-cloud-storage', '~> 1'
```

⚠️  Using `~> 1` means that it will pick any version above 1.x.x; so including major versions 2.y.y

The `~>` means "anything that uses this version or a higher value _of the last digit_ provided", so `~> 1.2.3` means "anything that is `1.2.x` with x≥3", and `~> 1.2` means "anything that is `1.x.*` with x≥2"… but `~> 1` means "anything that is `x.*.*` with x≥1".

That is likely not what we want, because that means it would include major versions bumps (thus breaking changes). So in practice, we're not really pinning anything when using `~> 1` since it might as well end up resolving to a version `3.2.1` which would not be compatible with our code anymore…

---

```suggestion
  spec.add_dependency 'google-cloud-storage', '~> 1.31'
```

+ run `bundle install` again to update `Gemfile.lock` accordingly

---

Fixed in c6a8f2c73e35cbf7d28e9712f9f6327e490c883b